src/misc/host_locale_posix.cpp

```diff
@@ -1247,7 +1247,7 @@ static std::vector<std::string> get_keyboard_layouts_tty()
 #endif // __linux__
 
 static std::vector<KeyboardLayoutMaybeCodepage> get_layouts_maybe_codepages_desktop(
-        std::string& log_info)
+        bool& is_layout_list_sorted, std::string& log_info)
```

Comment:
Consider using a struct for returning the values instead of a combination of an actual return value and out args. I always use this pattern these days in every language; no nore "out args" for me 😏 

```c
struct LayoutList {
   std::vector<KeyboardLayoutMaybeCodepage> layouts;
   std::string is_sorted;
   std::string log_info
}
```

But at the very least, prefix the out args with `out_`.

Reply:
For now just renamed the variables.

I'll need to do some more significant changes to the host-specific files, to support language preference lists (not just the GUI language), I'm going to experiment with the structures then. I'm just afraid I'll end up with a lot of structures used only once - bu we'll see.